apps/docs/src/app/core/component-docs/info-label/examples/info-label-text-icon-example.component.html

```diff
@@ -1,9 +1,10 @@
-<div style="margin-top: 10px" *ngFor="let item of ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10']">
+<div style="margin-top: 10px" *ngFor="let color of ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10']">
     <fd-info-label
         type="icon"
         glyph="hide"
         label="Info label"
-        [color]="item"
+        [color]="color"
         title="Info Label with Icon"
+        [ariaLabel]="'Info Label, type - icon, color scheme - ' + color"
```

Comment:
how would i18n work here?

Reply:
it is just example
Or what do you mean?